README.md

```diff
@@ -1 +1,3 @@
 Here is some text woot!
+
+#Here is more texts
```

Comment:
**_Why did you do this on line 3? Your method looks great but it could be shorter. Comment...._**

Reply:
What does review mean?